### PR TITLE
CODENVY-1829 Add ability to use ws-agent from main assembly

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
If Codenvy has been started with /assembly mount, it may contain ws-agent in assembly main (by default location lib/ws-agent.tar.gz). If CLI detects one, it will use this ws-agent.

### What issues does this PR fix or reference?
https://github.com/codenvy/codenvy/issues/1829

#### Changelog
The CLI can now use the ws-agent from a mounted assembly.

#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

File: dockerfiles/cli/scripts/cmd_config.sh [hunk 1]
```diff
@@ -73,6 +73,11 @@ generate_configuration_with_puppet() {
     WRITE_PARAMETERS=" -e \"PATH_TO_CHE_ASSEMBLY=${CHE_ASSEMBLY}\""
   fi
 
+  if [[ local_assembly && -f $(echo ${CHE_CONTAINER_ASSEMBLY_FULL_PATH}/lib/${WS_AGENT_ASSEMBLY}) ]]; then
+    info "Using ws-agent from main assembly"
```
Comment (TylerJewell) [inline dockerfiles/cli/scripts/cmd_config.sh]: info "config" "Using ws-agent from assembly"

Comment (mkuznyetsov) [inline dockerfiles/cli/scripts/cmd_config.sh]: fixed

**Comment by skabashnyuk** [comment]
@codenvy/pm  please approve

**Comment by bmicklea** [comment]
Will this need to be part of documentation for doing custom assemblies?

**Comment by mkuznyetsov** [comment]
don't think so, it is our internal mechanism for testing these custom assemblies

**Pull request merged**